src/core/src/sketch/lca_db.rs

```diff
@@ -0,0 +1,246 @@
+use std::fs::File;
```

Comment:
`sourmash::sketch` is aimed at sketches (like MinHash or Nodegraph), indices should go in `sourmash::index`. Move this file to `src/core/src/index/lca_db.rs`